### PR TITLE
Tiny update index page

### PR DESCRIPTION
**Pull request by Croden1999**

### CODE REVIEW

File: texts/index.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ texts.push([
 	"Quickstart", // 36
 	"Blog", // 37
 	"Not yet", // 38
+	"FAGs", // 39
```
Comment (bookmist) [inline texts/index.js:44]: G?

Comment (Croden1999) [reply texts/index.js]: sorry, my fault.

**Review by nishinji — APPROVED** [review]
I forgot to change `Blog` 🤦‍♂️

**Comment by Croden1999** [comment]
Mr.@nishinji, thanks.

**Pull request merged**